plugins/org.locationtech.udig.catalog.db2/src/org/locationtech/udig/catalog/db2/internal/messages.properties

```diff
@@ -15,5 +15,7 @@ DB2WizardPage_installDrivers=Install DB2 drivers from client
 DB2WizardPage_connectionTask=Attempting to make database connection
 DB2WizardPage_warning=An error will occur because the DB2 drivers were not loaded.  Please restart application
 DB2WizardPage_title=DB2 Wizard
+DB2WizardPage_button_lookup_text=Get schemas
```

Comment:
I've updated and committed the add DB2 features documentation.

You mentioned previously collapsing the commits - that is fine.

When do you think this can be integrated to master?